Codecademy/gamut
# fix(AppHeader): visual focus to menu descriptions

### Overview


before:
After opening "Catalog" menu focus is set to the "Popular course topics Explore free or paid courses in topics that interest you." text and focus indicator is not visible.
Other occurrences: Resources menu item.

after: 
After opening "Catalog" menu (or "Resources" menu) w/ keyboard, focus indicator should be visible.



### PR Checklist

- [X] Related to JIRA ticket: [DISC-899]

#### Testing Instructions

In preview env,
using keyboard, navigate to catalog and open dropdown. confirm visible outline around description.
repeat with resources dropdown. 

Now repeat both steps using mouse only, confirm you dont see any outline upon opening the dropdown menu




[Don't make me tap the sign.](https://i.imgur.com/sy93D9I.png)

#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Monolith     | [Monolith PR](https://github.com/codecademy-engineering/Codecademy/pull/33370)  | [Monolith Env](https://pr-33370-monolith.dev-eks.codecademy.com/) | 
| Portal       | [Portal Link](https://github.com/codecademy-engineering/mono/pull/536)  | [Portal Env](https://tengu.codecademy.com/catalog?PR_ENV=portal-app-pr-536)   | 




[DISC-899]: https://codecademy.atlassian.net/browse/DISC-899?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
